page-objects/lightingPage.ts

```diff
@@ -7,4 +7,27 @@ export class LightingPage extends HelperBase {
     constructor(page: Page){
         super(page)
     }
-}
\ No newline at end of file
+
+    async AdjustableWallLamp() {
+        const element = this.page.locator('div:nth-child(9) > div > a');
+        await element.click(); 
+    }
+
+    async ScrollToElement() {
+        const element = this.page.locator('div:nth-child(9) > div > a');
+        await element.scrollIntoViewIfNeeded();
+    }
+    async AddToElement(){
+        await this.page.locator('.description').getByText("Add to cart").click()
+    }
+    async Card(){
+        await this.page.locator('.menu-item-large').getByText("Cart (1)").click()
+    }
+    async Plus(){
```

Comment:
Function name should be more clear, and descriptive, so that user is aware what should be done in it.